runtime/src/lib.rs

```diff
@@ -534,6 +534,12 @@ impl pallet_scheduler::Config for Runtime {
 	type WeightInfo = weights::pallet_scheduler::WeightInfo<Runtime>;
 }
 
+impl pallet_utility::Config for Runtime {
+	type Event = Event;
+	type Call = Call;
+	type WeightInfo = ();
```

Comment:
Here you need to fill in the actual weight info, just like above on line 534.

You need to declare the `pallet_utility` module manually in the weights module.